project/Dependencies.scala

```diff
@@ -33,6 +34,7 @@ object Dependencies {
   val sparkVersion = sys.env.getOrElse("SPARK_VERSION", "1.5.2")
   lazy val sparkDeps = Seq(
     "org.apache.spark" %% "spark-core" % sparkVersion % "provided" excludeAll(excludeNettyIo, excludeQQ),
+    "org.apache.spark" %% "spark-sql" % sparkVersion % "provided" excludeAll(excludeNettyIo, excludeQQ),
```

Comment:
Why? Is it to include DataFrames?

Instead of pulling in spark-sql, I would say put the DataFrames logic in job-server-extras.